go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -87,24 +84,15 @@ func (npw *nodePortWatcher) AddService(service *kapi.Service) {
 					continue
 				}
 				flowProtocol := protocol
-				nw_dst := "nw_dst"
-				nw_src := "nw_src"
+				nwDst := "nw_dst"
 				if utilnet.IsIPv6String(ing.IP) {
 					flowProtocol = protocol6
-					nw_dst = "ipv6_dst"
-					nw_src = "ipv6_src"
+					nwDst = "ipv6_dst"
 				}
-				// Send to CT from external net with DNAT, eventually will land in table 1 (traffic to pods)
+				// Send directly to OVN. OVN GR will have load balancer for Ingress IP Service
```

Comment:
we can send it directly to the ClusterIP, that are the  OVN k8s-cluster-lb* , does it facilitates something?

Reply:
no the packet arrives at the GR which does not have the cluster ip load balancer on it, only node port, ingress, external IP. Doesn't really help us to add the cluster IP load balancer to the GR.